samples/Store/Domain/Domain.fsproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard2.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
this may not need net6.0 as it does not ref .Core ?

Reply:
I did a global search/replace. Is there a reason to keep supporting standard here?

Reply:
not really, but I was at one point trying to make it absolutely clear that the basic requirements of equinox (and of a well built Domain) are minimal
any more onerous requirements would be dictated by the specific store.
Ya know onions and hexagons
But I guess in today's world, `net6.0` is the new `netstandard2.0` ;)

Reply:
I agree with the point, it was lost on me

Reply:
Yeah, but it was far more relevant as a real world concern a few years back
Even making Equinox.Core cope with netstandard2.1 by sifting stuff around is only significant if there is a single store that can do something useful in some context due to not presenting a `net6.0` requirement.
But if Equinox, Domain and TodoBackend can stay 2.1 then they might as well

Reply:
Downgraded Eqx, Domain, and Domain.Test 👍